backend/src/main/java/com/crm/exception/user/LoginExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.crm.exception.user;
+
+import com.crm.exception.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.servlet.HandlerMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Collections;
+
+@RestControllerAdvice
+public class LoginExceptionHandler {
```

Comment:
Why create a separate class for exception handling when we have GlobalExceptionHandler? Was it done intentionally?

Reply:
This class catches only exceptions which inherit from CrmLoginException